docs/tutorial/part02.rst

```diff
@@ -1,6 +1,71 @@
-==========================
-Set the Neomodel database.
-==========================
+============================
+Set the NeoModel database
+============================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+On this section, we are going to install Neo4j on our local machine and show how you are going to connect it with our app.
+First, you need to download the desktop app of Neo4j_.
+
+    .. _neo4j: https://neo4j.com/download/
+
+Follow the steps of the wizard to install the app. Then you need to set your database; here is the official guide_ that
+you can follow to get Neo4j working on your local machine.
+
+    .. _guide: https://neo4j.com/developer/get-started/
+
+On the app example, we are going to use the LEAKS DATABASE. This is the data from the  Panama Papers, the Offshore Leaks,
+the Bahamas Leaks provided by the ICIJ(International Consortium of Investigative Journalists) here are some information about
+the ICIJ_ and the offshore_ leaks.
+
+    .. _icij: https://www.icij.org/about/
+    .. _offshore: https://offshoreleaks.icij.org/
+
+On this tutorial, we are going to concentrate on replicating the offshore leaks search app. It’s important that you know

```

Comment:
I believe the word focus suits better the context than concentrate. Honestly I don't understand the second sentence of this paragraph "It's important that you know the entities that we are going to handle the app." I think this should be rephrased it.